frontends/web/src/components/accountSelector/accountselector.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2022 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useTranslation } from 'react-i18next';
+import { AccountCode } from '../../api/account';
+import { Button, Select } from '../forms';
+import style from './accountselector.module.css';
+
+type TOption = {
+    text: string;
+    value: AccountCode;
+}
+
+type TAccountSelector = {
+    title: string;
+    options: TOption[];
+    selected?: string;
+    onChange: (value: string) => void;
+    onProceed: () => void;
+}
+export const AccountSelector = ({ title, options, selected, onChange, onProceed }: TAccountSelector) => {
+  const { t } = useTranslation();
+  return (
+    <div className="content narrow isVerticallyCentered">
+      <h1 className={style.title}>{title}</h1>
```

Comment:
I am not sure about the title repeating here. lets discusss dm

Reply:
instead of using a new custom title could you use `class="subTitle"` (should be centered if you use `<View>` component lets check together tomorrow maybe or next week).

Reply:
`.subtitle` has a different styling (heavier font): 

<img width="291" alt="Screenshot 2023-01-26 at 07 50 51" src="https://user-images.githubusercontent.com/28676406/214774456-6fe6b0c2-1845-4e45-86c1-9d8f92be807b.png">

intentional (do we wanna use it here)?

Reply:
looks like `subtitle` has a different style (smaller font):

<img width="160" alt="Screenshot 2023-01-26 at 07 52 05" src="https://user-images.githubusercontent.com/28676406/214774632-3d350490-b00d-401c-aa2a-044be3639462.png">
intentional / do we actually wanna use smaller font here?   

Moreover, the `View`  component doesn't have sufficient styling to achieve what we want in this page. We need it vertically centred and has a narrow size (max-width to be of certain size, check `.content.narrow`) - and `View` has neither of these options. 

Since we plan to use View everywhere and dropping the container divs, shall I work on `View` first to allow for what we need, and then use it here?

Reply:
subTitle is used in many components. in the same file (layout.css) there is also `.title` you can try that too, but I don't see where it is used in the codebase, hard to search for.

Reply:
I think `subTitle` is fine, it gets bigger on large screen. also ViewHeader uses the same font size through CSS variable, `var(--size-subheader)`. please have a look at passphrase.tsx (under device-settings go to enable passphrase, there is some titles)

Reply:
Yes, `.title` is the exact class for this I think :)